reader/src/main/java/org/jline/reader/impl/history/DefaultHistory.java

```diff
@@ -100,12 +101,19 @@ public void load() throws IOException {
     protected void addHistoryLine(Path path, String line) {
         if (reader.isSet(LineReader.Option.HISTORY_TIMESTAMPED)) {
             int idx = line.indexOf(':');
+            final String badHistoryFileSyntax = "Bad history file syntax! " +
+                "The history file `" + path + "` may be an older history: " +
+                "please remove it or use a different history file.";
             if (idx < 0) {
-                throw new IllegalArgumentException("Bad history file syntax! " +
-                        "The history file `" + path + "` may be an older history: " +
-                        "please remove it or use a different history file.");
+                throw new IllegalArgumentException(badHistoryFileSyntax);
             }
-            Instant time = Instant.ofEpochMilli(Long.parseLong(line.substring(0, idx)));
+            Instant time;
+            try {
+              time = Instant.ofEpochMilli(Long.parseLong(line.substring(0, idx)));
```

Comment:
Could you please change the 2 spaces indentation in a 4 spaces indentation ? That way, I'll merge the PR directly. Thx !

Reply:
Done, please let me know if I need to squash

Reply:
Yes, please I'd rather have a single commit for that.

Reply:
ok, done